.github/workflows/update-packages-from-list.yml

```diff
@@ -73,6 +75,6 @@ jobs:
           branch: buildbot/automatic-package-updates
```

Comment:
Question that I have here is what would be the desired outcome when the weekly package updates PR is not handled within the week:
- create another PR next Monday (would need a differently named branch too)
- update the existing PR (stays as is)

What do you think?

Reply:
I would prefer to have another PR for a new week.
If we would explicitly want to merge new updates into an existing PR, then we could simply close the existing one, and run new Github Actions.

Reply:
Done